mail_tracking/models/mail_tracking_email.py

```diff
@@ -163,16 +160,14 @@ def _compute_recipient_address(self):
             else:
                 email.recipient_address = email.recipient
 
-    @api.multi
     @api.depends('name', 'recipient')
-    def _compute_display_name(self):
+    def _comp_display_name(self):
```

Comment:
I renamed this function since it triggered a warning that the field depended on itself.
The `_compute_display_name` function in models.py adds `display_name` as a dependency, so to avoid this warning I had to rename the function...